Failing api-integration-tests

https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/kubeflow_pipelines/3536/kubeflow-pipeline-e2e-test/1251074359473213440

```
run-api-integration-tests:	=== RUN   TestRunApi/TestRunApis
run-api-integration-tests:	--- FAIL: TestRunApi (2.93s)
run-api-integration-tests:	    --- FAIL: TestRunApi/TestRunApis (2.67s)
run-api-integration-tests:	        run_api_test.go:295: 
run-api-integration-tests:	            	Error Trace:	run_api_test.go:295
run-api-integration-tests:	            	            				run_api_test.go:108
run-api-integration-tests:	            	Error:      	Not equal: 
run-api-integration-tests:	            	            	expected: &run_model.APIRun{CreatedAt:strfmt.DateTime{wall:0x0, ext:63722713408, loc:(*time.Location)(nil)}, Description:"this is hello world", Error:"", FinishedAt:strfmt.DateTime{wall:0x0, ext:62135596800, loc:(*time.Location)(nil)}, ID:"93b1f366-002a-4634-a58a-230cd9fca6a5", Metrics:[]*run_model.APIRunMetric(nil), Name:"hello world", PipelineSpec:(*run_model.APIPipelineSpec)(0xc0009e2900), ResourceReferences:[]*run_model.APIResourceReference{(*run_model.APIResourceReference)(0xc0007faab0), (*run_model.APIResourceReference)(0xc0007faae0)}, ScheduledAt:strfmt.DateTime{wall:0x0, ext:62135596800, loc:(*time.Location)(nil)}, Status:"", StorageState:""}
run-api-integration-tests:	            	            	actual  : &run_model.APIRun{CreatedAt:strfmt.DateTime{wall:0x0, ext:63722713408, loc:(*time.Location)(nil)}, Description:"this is hello world", Error:"", FinishedAt:strfmt.DateTime{wall:0x0, ext:62135596800, loc:(*time.Location)(nil)}, ID:"93b1f366-002a-4634-a58a-230cd9fca6a5", Metrics:[]*run_model.APIRunMetric(nil), Name:"hello world", PipelineSpec:(*run_model.APIPipelineSpec)(0xc0009e2000), ResourceReferences:[]*run_model.APIResourceReference{(*run_model.APIResourceReference)(0xc0003ea660), (*run_model.APIResourceReference)(0xc0003ea6f0)}, ScheduledAt:strfmt.DateTime{wall:0x0, ext:62135596800, loc:(*time.Location)(nil)}, Status:"", StorageState:""}
run-api-integration-tests:	            	            	
run-api-integration-tests:	            	            	Diff:
run-api-integration-tests:	            	            	--- Expected
run-api-integration-tests:	            	            	+++ Actual
run-api-integration-tests:	            	            	@@ -18,2 +18,10 @@
run-api-integration-tests:	            	            	    Key: (*run_model.APIResourceKey)({
run-api-integration-tests:	            	            	+    ID: (string) (len=36) "c6bf3773-faac-41ce-b651-d596a9fe7b6f",
run-api-integration-tests:	            	            	+    Type: (run_model.APIResourceType) (len=16) "PIPELINE_VERSION"
run-api-integration-tests:	            	            	+   }),
run-api-integration-tests:	            	            	+   Name: (string) (len=19) "hello-world-version",
run-api-integration-tests:	            	            	+   Relationship: (run_model.APIRelationship) (len=7) "CREATOR"
run-api-integration-tests:	            	            	+  }),
run-api-integration-tests:	            	            	+  (*run_model.APIResourceReference)({
run-api-integration-tests:	            	            	+   Key: (*run_model.APIResourceKey)({
run-api-integration-tests:	            	            	     ID: (string) (len=36) "f9b7354c-7531-456e-8cce-e72267b4325e",
run-api-integration-tests:	            	            	@@ -23,10 +31,2 @@
run-api-integration-tests:	            	            	    Relationship: (run_model.APIRelationship) (len=5) "OWNER"
run-api-integration-tests:	            	            	-  }),
run-api-integration-tests:	            	            	-  (*run_model.APIResourceReference)({
run-api-integration-tests:	            	            	-   Key: (*run_model.APIResourceKey)({
run-api-integration-tests:	            	            	-    ID: (string) (len=36) "c6bf3773-faac-41ce-b651-d596a9fe7b6f",
run-api-integration-tests:	            	            	-    Type: (run_model.APIResourceType) (len=16) "PIPELINE_VERSION"
run-api-integration-tests:	            	            	-   }),
run-api-integration-tests:	            	            	-   Name: (string) (len=19) "hello-world-version",
run-api-integration-tests:	            	            	-   Relationship: (run_model.APIRelationship) (len=7) "CREATOR"
run-api-integration-tests:	            	            	   })
run-api-integration-tests:	            	Test:       	TestRunApi/TestRunApis
run-api-integration-tests:	=== RUN   TestUpgrade
run-api-integration-tests:	=== RUN   TestUpgrade/TestPrepare
run-api-integration-tests:	=== RUN   TestUpgrade/TestVerify
run-api-integration-tests:	--- PASS: TestUpgrade (6.78s)
run-api-integration-tests:	    --- PASS: TestUpgrade/TestPrepare (6.44s)
run-api-integration-tests:	    --- PASS: TestUpgrade/TestVerify (0.03s)
run-api-integration-tests:	=== RUN   TestVisualizationAPI
run-api-integration-tests:	=== RUN   TestVisualizationAPI/TestVisualizationAPI
run-api-integration-tests:	--- PASS: TestVisualizationAPI (0.43s)
run-api-integration-tests:	    --- PASS: TestVisualizationAPI/TestVisualizationAPI (0.43s)
run-api-integration-tests:	FAIL
run-api-integration-tests:	FAIL	github.com/kubeflow/pipelines/backend/test/integration	108.236s
```

--- a/backend/test/integration/run_api_test.go
+++ b/backend/test/integration/run_api_test.go
@@ -2,6 +2,7 @@ package integration
 
 import (
 	"io/ioutil"
+	"reflect"
 	"testing"
 	"time"
 
@@ -263,7 +264,7 @@ func (s *RunApiTestSuite) checkTerminatedRunDetail(t *testing.T, runDetail *run_
 		ScheduledAt: runDetail.Run.ScheduledAt,
 		FinishedAt:  runDetail.Run.FinishedAt,
 	}
-	assert.Equal(t, expectedRun, runDetail.Run)
+	assert.True(t, reflect.DeepEqual(expectedRun, runDetail.Run))
 }
 
 func (s *RunApiTestSuite) checkHelloWorldRunDetail(t *testing.T, runDetail *run_model.APIRunDetail, experimentId string, experimentName string, pipelineVersionId string, pipelineVersionName string) {
@@ -292,7 +293,7 @@ func (s *RunApiTestSuite) checkHelloWorldRunDetail(t *testing.T, runDetail *run_
 		ScheduledAt: runDetail.Run.ScheduledAt,
 		FinishedAt:  runDetail.Run.FinishedAt,
 	}
-	assert.Equal(t, expectedRun, runDetail.Run)
+	assert.True(t, reflect.DeepEqual(expectedRun, runDetail.Run))
 }
 
 func (s *RunApiTestSuite) checkArgParamsRunDetail(t *testing.T, runDetail *run_model.APIRunDetail, experimentId string, experimentName string) {
@@ -323,7 +324,7 @@ func (s *RunApiTestSuite) checkArgParamsRunDetail(t *testing.T, runDetail *run_m
 		ScheduledAt: runDetail.Run.ScheduledAt,
 		FinishedAt:  runDetail.Run.FinishedAt,
 	}
-	assert.Equal(t, expectedRun, runDetail.Run)
+	assert.True(t, reflect.DeepEqual(expectedRun, runDetail.Run))
 }
 
 func TestRunApi(t *testing.T) {

